### ISSUE TITLE
Hide items from Autocomplete once selected

### ISSUE THREAD

**Issue by yarnball** [mvader/react-categorized-tag-input#19]
Hi,

When I add an "tag" item from the autocomplete, it does not get removed from the autocomplete list. 

This means it allows duplicates of the same tag to be created.

Is this a bug? Or is there a way to improve this?

**Comment by thienanle** [comment]
Yes, it would be great if this could be an option passed in as a prop.

**Comment by ehartford** [comment]
+1

**Comment by erizocosmico** [comment]
I can add a `allowDuplicates` prop to enable/disable this. Since it would be a breaking change, it would be `true` by default. Or maybe I will release a new major update. Sorry for not replying for a long time, completely forgot about this.

**Comment by yarnball** [comment]
All good- has your most recent update achieved this?

**Comment by erizocosmico** [comment]
Nope, has not been done yet.
